composer.json

```diff
@@ -1,4 +1,12 @@
 {
+    "name"       : "alleyinteractive/apple-news",
+    "description": "Apple News delivery for WordPress.",
```

Comment:
Please change to "Enable your WordPress blog content to be published to your Apple News channel."